src/app/personside/visittkort-v2/visittkort-utils.ts

```diff
@@ -70,3 +70,17 @@ export function hentPeriodeTekst(gyldighetstidspunkt: string | null, opphorstids
 
     return `${fom} ${ENDASH} ${tom}`;
 }
+
+export function mapUgyldigGT(gt: string): string {
+    if (gt === '0301') {
+        return 'Oslo';
+    } else if (gt === '4601') {
+        return 'Bergen';
+    } else if (gt === '5001') {
+        return 'Trondheim';
+    } else if (gt === '1103') {
+        return 'Stavanger';
+    } else {
+        return 'Ukjent';
+    }
+}
```

Comment:
Bare ett alternativ for tilfeller hvor listen kanskje oppdateres oftere (ikke så relevant her):
```suggestion
const ugyldig_gt_map = {
    '0301': 'Oslo',
    '4601': 'Bergen',
    '5001': 'Tronheim',
    '1103': 'Stavanger',
};
export function mapUgyldigGT(gt: string): string {
    return ugyldig_gt_map[gt] ?? 'Ukjent';
}
```